server/datamodel.go

```diff
@@ -828,6 +828,11 @@ func NoErrExplicitTs(id, topic string, serverTs, incomingReqTs time.Time) *Serve
 	return NoErrParamsExplicitTs(id, topic, serverTs, incomingReqTs, nil)
 }
 
+// NoErrReply indicates successful completion as a reply to a client message (200).
+func NoErrReply(msg *ClientComMessage, ts time.Time) *ServerComMessage {
+	return NoErrExplicitTs(msg.Id, msg.Original, msg.Timestamp, ts)
```

Comment:
I think it should be `ts, msg.Timestamp` like everywhere else. The `incomingReqTs` is `msg.Timestamp`, `serverTs` is `ts`, right?